src/devices/src/virtio/net/device.rs

```diff
@@ -31,7 +31,9 @@ use virtio_gen::virtio_net::{
     VIRTIO_NET_F_GUEST_TSO4, VIRTIO_NET_F_GUEST_UFO, VIRTIO_NET_F_HOST_TSO4, VIRTIO_NET_F_HOST_UFO,
     VIRTIO_NET_F_MAC,
 };
-use vm_memory::{ByteValued, Bytes, GuestAddress, GuestMemoryError, GuestMemoryMmap};
+use vm_memory::{
+    ByteValued, Bytes, GuestAddress, GuestMemoryError, GuestMemoryMmap, VolatileMemory,
```

Comment:
I can't figure out where `VolatileMemory` is used..

Reply:
It's a leftover. Removed it.